taskcluster/utils.py

```diff
@@ -353,4 +411,14 @@ def do_GET(self):
 
   while not creds[0]:
     server.handle_request()
+
+  if sys.stdout.isatty():
+      print "Retreived credentials successfully.\n\nType 'yes' to save to " + CONFIG_FILE
+      a = raw_input().strip().lower()
+      if a == 'yes':
+        print "Saving configuration file"
+        writeCfgFile(creds[0])
+      else:
+        print "Not saving configuration file"
```

Comment:
Okay, a crazy idea would be to use the webserver to do a confirm() in javascript to see if we should save the configuration file...  of course we need to run `server.handle_request()` until we got the credentials and the browser have told us if the credentials should be saved... it's an option.. maybe we can leave it for later...

Probably a bit overkill..  he he :)